\section{Methods}
% Structure with subsections
% Methodology aligns with research question (appropriate resources/methods used)
% General description of the methodology
% Enough details to ensure the reproducibility of results
The following structured approach will be pursued to answer the research question:

\paragraph{Analysis of Differentially Expressed Genes (DEGs).}\label{sec:methods-deg}
The used dataset of gene expression profiles from dermal fibroblasts from patients with hEDS and healthy controls is available at the NCBI GEO database with the accession number GSE218012 \cite{Ritelli2020}. The analysis is performed with the R-packages DeSeq2 (v1.40.2) and limma (v.3.56.2) based on the R-Script from GEO2R to identify up-regulated and down-regulated genes \cite{DESeq2, limma}. Genes with a log2-fold change $> \pm 0.5$ and a by the Benjamin-Hochberg procedure adjusted p-value $< 0.05$ are included. The cut-offs were chosen based on similar research \cite{Karimizadeh2019, Lim2019}.

\paragraph{Network Creation.}
The Protein-protein interaction (PPI) network is created in Cytoscape (v3.10.1) \cite{Cytoscape} by querying the before-identified DEGs from the STRING database with a medium confidence interaction score ($> 0.4$) \cite{StringDB}. Since hEDS belongs to the family of Ehlers-Danlos syndromes, its molecular cause is most likely closely related to other EDS types. Therefore, the network is expanded by additionally querying genes related to other EDS types retrieved from Disease ontology (Disease Ontology ID 13359) \cite{DO}. The resulting network is annotated with the differential expression data.
	
\paragraph{Gene Ontology and Clustering.}
GeneOntology (GO) enrichment is performed using the R-package clusterProfiler (v4.8.3) to gain insight into biological processes and molecular functions affected by DEGs, with results with $p < 0.05$ being considered significant \cite{Ashburner2000,Consortium2023, Wu2021}. To attain more detailed insights into specific parts, the created network is clustered using two different algorithms: MCODE and Community Clustering. MCODE is a clustering algorithm designed to find highly connected regions in PPI networks that might represent molecular complexes \cite{mcode}. On the other hand, GLay, a community clustering algorithm, is designed for a functional interpretation of biological processes and pathways. Clustering is performed in Cytoscape using the clusterMaker2 app (v2.3.4) with the default parameters \cite{Cytoscape, clusterMaker2}. Only clusters of more than 15 genes are considered to ensure relevance and keep the analysis feasible. Analysis of the clusters includes investigating whether genes are clustered with genes that cause other EDS types and whether the clusters consist of up-regulated or down-regulated genes or a combination of both. Heat Diffusion is applied on larger clusters to identify genes closely connected to EDS genes not captured by smaller clusters, starting with the EDS nodes using Cytoscape functionality \cite{heatDiffusion}, using a time parameter of $t=0.3$.


%[TODO: position of eds genes in network - why]