\section{Methods}
% Structure with subsections
% Methodology aligns to research question (appropriate resources/methods used)
% General description of the methodology
% Enough details to ensure reproducibility of results

\subsection{Analysis of Differentially Expressed Genes and Network Creation}\label{sec:methods-deg}
A dataset of gene expression profiles from dermal fibroblasts from patients with hEDS and healthy controls is used. It is available at the NCBI GEO database with the accession number GSE218012 \cite{Ritelli2020}. The analysis is performed with the R-package DeSeq2 based on the analysis script exported from GEO2R to identify up-regulated and down-regulated genes \cite{DESeq2}. Genes with a log2-fold change $> \pm 0.5$ and a p-value $< 0.05$ after adjustment based on False-Discovery-Rate using the Benjamin-Hochberg procedure where considered. The cut-offs where chosen based on similar research \cite{Karimizadeh2019, Lim2019}.

% Network creation
\begin{itemize}
	\item use Cytoscape to create network \cite{Cytoscape}
	\item query differentially expressed genes from string db \cite{StringDB} (confidence cut off 0.4)
	\item query eds genes related to other eds types additionally: 21 genes retrieved from Disease Ontology with Disease Ontology ID 13359 \cite{DO}, queried from string, again with a medium confidence cut off of 0.4
	\item load data about differential expression into network, for EDS genes and differentially expressed genes
\end{itemize}

\subsection{Enrichment analysis and clustering}
\begin{itemize}
	\item exploratory enrichment analysis on whole network
	\item use R-package clusterProfiler \cite{Wu2021} for enrichment
	\item large network, thus cluster before to get better insights for specific parts
	\item two different cluster methods used with different resulting cluster structure
	\item create subnetworks for clusters with more than 15 nodes [TODO: why this threshold? to focus on more relevant because many in MCODE between 10 and 15 nodes, for community clustering smaller ones probably found by MCODE]
\end{itemize}
\subsubsection{MCODE}
\begin{itemize}
	\item MCODE finds "densely connected regions in large protein-protein interaction networks that may represent molecular complexes" \cite{mcode}
	\item results in smaller clusters
	\item suited to analyse molecular function
	\item use default parameters
	\item analysis of molecular function with clusterProfiler with GO over-representation analysis
	\item do we see clusters with other molecular functions than expected
	\item are genes clustered together with other eds genes
	\item are clusters generelly mostly upregulated or downregulated
\end{itemize}
\subsubsection{Community Clustering}
\begin{itemize}
	\item small cluster not helpful for biological processes and pathways
	\item therefore use second clustering method
	\item community clustering with GLay, "more suitable for functional interpretation" \cite{GLay}
\end{itemize}